src/Controller/SubscriptionController.php

```diff
@@ -84,7 +99,7 @@ public function subscribe($entity_type_id, EntityInterface $group, OgMembershipT
       throw new AccessDeniedHttpException();
     }
 
-    $user = User::load($this->currentUser()->id());
+    $user = $this->entityTypeManager->load($this->currentUser()->id());
```

Comment:
Here instead we can call `$this->getEntityTypeManager()->getStorage('user')->load()`.